### PR TITLE
perf(core): preserve ops between snapshots

### PR DESCRIPTION
**Pull request by bartlomieju**
This commit changes the build process in a way that preserves already registered ops
in the snapshot. This allows us to skip creating hundreds of "v8::String" on each
startup, but sadly there is still some op registration going on startup (however we're
registering 49 ops instead of >200 ops). This situation could be further improved,
by moving some of the ops from "runtime/" to a separate extension crates.

### CODE REVIEW

**Comment by bartlomieju** [comment]
Blocked by https://github.com/denoland/rusty_v8/pull/1197

File: ext/node/polyfills/internal/readline/utils.mjs [hunk 1]
```diff
@@ -38,10 +38,10 @@ export function CSI(strings, ...args) {
...
 
 CSI.kEscape = kEscape;
-CSI.kClearToLineBeginning = CSI`1K`;
-CSI.kClearToLineEnd = CSI`0K`;
-CSI.kClearLine = CSI`2K`;
-CSI.kClearScreenDown = CSI`0J`;
+CSI.kClearToLineBeginning = `${kEscape}[1K`;
+CSI.kClearToLineEnd = `${kEscape}[0K`;
+CSI.kClearLine = `${kEscape}[2K`;
+CSI.kClearScreenDown = `${kEscape}[0J`;
```
Comment (bartlomieju) [inline ext/node/polyfills/internal/readline/utils.mjs:44]: These caused strange errors during snapshotting and arguably it's cleared what's going on here now

**Comment by bartlomieju** [comment]
Blocked by https://github.com/denoland/deno/pulls

**Review by littledivy — APPROVED** [review]
LGTM!

**Pull request merged**